src/main/entities/CustomPassword.java

```diff
@@ -0,0 +1,16 @@
+package entities;
+
+public class CustomPassword extends Password {
+    StrengthChecker strengthChecker = new StrengthChecker();
```

Comment:
Should a password be checking its own strength?

Reply:
As per the design, `CustomPassword` has the role of getting a strength for itself and `NewPassword` has the role of getting criteria for itself.